chapter-7.qmd

```diff
@@ -0,0 +1,11 @@
+---
+title: "Workflow: scripts and projects"
+---
+
+
+
+## Solutions
+
+1. What tip from twitter did i like?
```

Comment:
Remove the text here, just state the answer.

Reply:
i would think a future person would want to see the questions with the answers